src/Tribe/Main.php

```diff
@@ -98,81 +98,39 @@ public function init_libraries() {
 	/**
 	 * Registers resources that can/should be enqueued
 	 */
-	public function register_resources() {
-		$resources_url = plugins_url( 'src/resources', dirname( dirname( __FILE__ ) ) );
-
-		wp_register_style(
-			'tribe-common-admin',
-			$resources_url . '/css/tribe-common-admin.css',
-			array(),
-			apply_filters( 'tribe_events_css_version', self::VERSION )
-		);
-
-		wp_register_script(
-			'ba-dotimeout',
-			$resources_url . '/js/jquery.ba-dotimeout.js',
-			array(
-				'jquery',
-			),
-			apply_filters( 'tribe_events_css_version', self::VERSION ),
-			true
-		);
-
-		wp_register_script(
-			'tribe-inline-bumpdown',
-			$resources_url . '/js/inline-bumpdown.js',
-			array(
-				'ba-dotimeout',
-			),
-			apply_filters( 'tribe_events_css_version', self::VERSION ),
-			true
-		);
-
-		wp_register_script(
-			'tribe-notice-dismiss',
-			$resources_url . '/js/notice-dismiss.js',
-			array( 'jquery' ),
-			apply_filters( 'tribe_events_css_version', self::VERSION ),
-			true
-		);
-	}
-
-	/**
-	 * Registers vendor assets that can/should be enqueued
-	 */
-	public function register_vendor() {
-		$vendor_base = plugins_url( 'vendor', dirname( dirname( __FILE__ ) ) );
-
-		wp_register_style(
-			'tribe-jquery-ui-theme',
-			$vendor_base . '/jquery/ui.theme.css',
-			array(),
-			apply_filters( 'tribe_events_css_version', self::VERSION )
-		);
-
-		wp_register_style(
-			'tribe-jquery-ui-datepicker',
-			$vendor_base . '/jquery/ui.datepicker.css',
-			array( 'tribe-jquery-ui-theme' ),
-			apply_filters( 'tribe_events_css_version', self::VERSION )
-		);
-
+	public function load_assets() {
+		// These ones are only registred
+		tribe_assets( $this, array(
+			array( 'ba-dotimeout', 'jquery.ba-dotimeout.js', array( 'jquery' ) ),
+			array( 'tribe-jquery-ui-theme', 'vendor/jquery/ui.theme.css' ),
+		) );
```

Comment:
The formatting of this `tribe_assets()` call is kind of hard to read. Maybe:

```php
tribe_assets( $this, array(
  array(
    'ba-dotimeout',
    'jquery.ba-dotimeout.js',
    array( 'jquery' ),
  ),
  array(
    'tribe-jquery-ui-theme',
    'vendor/jquery/ui.theme.css'
  ),
) );
```

Though maybe it'd be easier still with two distinct `tribe_asset()` calls:

```php
tribe_asset( $this, 'ba-dotimeout', 'jquery.ba-dotimeout.js', array( 'jquery' ) );
tribe_asset( $this, 'tribe-jquery-ui-theme', 'vendor/jquery/ui.theme.css' );
```

Rinse and repeat this comment for the next `tribe_assets()` call.